src/components/projects_home/projectsCard.tsx

```diff
@@ -1,178 +1,159 @@
-        }
-        else {
-            return (
-                <span className={style['icon-no-show']}>
-                    {props.title.slice(0, 1)}
-                </span>
-            )
-        }
+  console.log(props.contributors);
+  const linkStyle = {
+    "text-decoration": "none",
+    color: "black",
+  };
+
+  function renderIconOrRenderFirstLetterInName(icon_url: any) {
+    const backgroundStyle = {
+      backgroundImage: `url(${icon_url})`,
+    };
+    if (icon_url != null) {
+      return (
+        <span style={backgroundStyle} className={style["project-logo"]}></span>
+      );
+    } else {
+      return (
+        <span className={style["icon-no-show"]}>{props.title.slice(0, 1)}</span>
+      );
     }
-    console.log(props.contributors[2])
-    // let contributors = props.contributors[5]
-    // let lengthOfContributors = props.contributors.length
-    // let subtractedLength = contributors.length - lengthOfContributors
-
-    function renderAvatarOrFirstLetterInName(contributor: any) {
-        const contributorBackGround = {
-            'backgroundImage': `url(${contributor.profile_image})`
-        }
-        if (contributor.profile_image != null) {
-            return (
-                <div style={contributorBackGround} className={style['contributor-avatar']}></div>
-            )
-        }
-
-    }
-    let theme = window.localStorage.getItem('theme');
-    if (theme == 'dark') {
-        let card = document.querySelector(style['project-card-body']);
-        console.log(card)
-        card?.classList.add('project-card-body-dark')
-        window.localStorage.setItem('theme', 'dark')
-    }
-    else {
-        document.body.classList.remove('dark-body');
-        window.localStorage.setItem('theme', 'light')
-    }
-
-    const widthStyle = {
-        'width': `${props.tasksCompleted}%`
+  }
+  console.log(props.contributors[2]);
+  // let contributors = props.contributors[5]
+  // let lengthOfContributors = props.contributors.length
+  // let subtractedLength = contributors.length - lengthOfContributors
+
+  function renderAvatarOrFirstLetterInName(contributor: any) {
+    const contributorBackGround = {
+      backgroundImage: `url(${contributor.profile_image})`,
+    };
+    if (contributor.profile_image != null) {
+      return (
+        <div className={style["tooltip"]}>
+          <div
+            style={contributorBackGround}
+            className={style["contributor-avatar"]}
+          >
+            <span className={style["tooltip-text"]}>
+              {contributor.user_email}
+            </span>
+          </div>
+        </div>
+      );
+    } else {
+      return (
+        <div className={style["tooltip"]}>
+          <div className={style["no-image"]}>
+            {contributor.user_email.slice(0, 1)}
+
+            <span className={style["tooltip-text"]}>
+              {contributor.user_email}
+            </span>
+          </div>
+        </div>
+      );
     }
-    return (
```

Comment:
Removed old logic for trying to display the contributors avatar in the `projectsCard`